### PR TITLE
Remove telemetry override for CodeWhisperer

### PR DESCRIPTION
**Pull request by andrewyuq**
JB change: https://github.com/aws/aws-toolkit-jetbrains/pull/3211
aws-toolkit-common: https://github.com/aws/aws-toolkit-common/pull/338

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by justinmk3** [comment]
next step is to also  bump the aws/telemetry package to latest version: https://www.npmjs.com/package/@aws-toolkits/telemetry

File: package-lock.json [hunk 1]
```diff
@@ -1,14278 +1,8 @@
 {
     "name": "aws-toolkit-vscode",
     "version": "1.47.0-SNAPSHOT",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```
Comment (justinmk3) [inline package-lock.json:4]: This is not correct. Please update your local `npm`.

**Comment by andrewyuq** [comment]
> next step is to also bump the aws/telemetry package to latest version: https://www.npmjs.com/package/@aws-toolkits/telemetry
> 
> https://github.com/aws/aws-toolkit-vscode/blob/a3735b9ea98cae6dc9e1cb2a1a27e407db1752d5/package.json#L3170

I have updated npm to v18.7.0(latest) and redo npm install, please check if the package-lock.json looks correct

File: package-lock.json (hunk 1 above)
Comment (andrewyuq) [reply package-lock.json]: I have updated npm to v18.7.0(latest) and redo npm install, please check if the package-lock.json looks correct

**Pull request merged**